config/kube-core/config.go

```diff
@@ -50,3 +51,9 @@ type NetworkConfig struct {
 type InitialConfig struct {
 	InitialMasterCount int `json:"initial_master_count"`
 }
+
+// ClusterConfig holds global/general information about the cluster.
+type ClusterConfig struct {
+	Name             string `json:"name"`
+	APIServerAddress string `json:"api_server_address"`
```

Comment:
How is this different from `AdvertiseAddress` in `NetworkConfig`?

That one is currently filled by the following tectonic variable:

https://github.com/coreos/tectonic-installer/blob/master/modules/bootkube/variables.tf#L1

Reply:
Advertise address is an IP address. This seems to be set to "0.0.0.0" by default. @aaronlevy 

We need `kube_apiserver_url` https://github.com/coreos/tectonic-installer/blob/master/platforms/aws/tectonic.tf#L22

https://github.com/coreos/tectonic-installer/blob/master/modules/bootkube/variables.tf#L81-L84

Reply:
ok then I would rename this to APIServerURL

Reply:
renamed to `APIServerURL`